common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -60,48 +56,14 @@ protected void init(
      *
      * <p>Returns whether the feature was successfully activated
      */
-    public final boolean enable() {
+    public final void enable() {
         if (enabled)
```

Comment:
I realize this is not new code, but I really really dislike the use of multiline if statements without braces. I'd like to vote for having a formal rule against it; preferably checked by style checkers if possible...

Reply:
I’m fine either way, get a consensus and I’ll follow the guideline